src/bmi_lgar.cxx

```diff
@@ -65,6 +64,15 @@ Update()
   if (state->lgar_bmi_params.sft_coupled)
     frozen_factor_hydraulic_conductivity(state->lgar_bmi_params);
 
+  double volchange_calib_cm = 0.0;
+
+  if(state->lgar_bmi_params.calib_params_flag) {
+    volchange_calib_cm = update_calibratable_parameters(); // change in soil water volume due to calibratable parameters
+    state->lgar_bmi_params.calib_params_flag = false;
+  }
+  
+  assert(state->lgar_bmi_params.calib_params_flag == false);
```

Comment:
This assert isn't needed, there is no way to get to this statement without `state->lgar_bmi_params.calib_params_flag` being `false`

Reply:
good catch, agree and will remove the assertion. 

I have tested it in the `ngen` framework and the model picked the right values provided in the `model_params` block, but if something appears to be missing/needed, will definitely include it.